project/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.forms.models import ModelForm
+from tracker.settings import TIME_ZONE
+
+
+class User(AbstractUser):
+    def __repr__(self):
+        return f"<User username={self.username}>"
+
+    def __str__(self):
+        return self.username
+
+
+class habitEntry(models.Model):
+        user = models.ForeignKey(to=User, on_delete=models.CASCADE, related_name="entry")
+        habit = models.CharField(max_length=255)
```

Comment:
You'll need a habit model and a ForeignKey here, so that you can track the same habit with each entry